roles/usb_lib/files/upload2usb/upload2usb.php

```diff
@@ -31,22 +33,12 @@ function getTargetUSBDriveLocation () {
          } elseif ($rmv_usb_path_count > 1) {
```

Comment:
```suggestion
         } elseif ($rmv_usb_path > 1) {
```

Reply:
FWIW `count(explode("\n", $rmv_usb_path))` conflates 0 with 1, and gives the wrong number in general (off by 1) as this PR currently trims any trailing `\n` back on line 22:

`$rmv_usb_path = trim(shell_exec('lsblk -no MOUNTPOINT | grep "^/media"'));` 

My Opinion: @avni can/should greatly simplify, and clean up all this fenceposting confusion, starting here:

https://github.com/iiab/iiab/pull/4097#issuecomment-3345744455

Reply:
> BUT I still would be surprised that multiple mountpoints, even on `vfat`, are the cause of any troubles. I haven't seen any hard evidence yet.

#4066 comes to mind, and cruse the issues this is a long standing known issue.